tests/flow/test_keyspace_accesses.py

```
@@ -41,6 +41,12 @@ def test01_graph_access_on_invalid_key(self):
             assert("WRONGTYPE" in str(e))
             pass
 
+        try:
+            self.redis_con.execute_command("GRAPH.DELETE", "integer_key")
+            assert(False)
```

Ruff (0.15.14)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)